src/components/UploadSong.css

```diff
@@ -0,0 +1,60 @@
+.boxStyle {
+    display: flex;
+    width: 100vw;
+    align-items: center;
```

Comment:
Visual-music uses module.scss not pure css. So rename your file to UploadSong.module.scss and nest elements together no more when 3 levels due to performance reasons

Reply:
Done.